util_linux.c

```diff
@@ -15,10 +15,13 @@ int32_t get_desktop_env(void) {
             return DESKTOP_ENV_GNOME3;
         } else if (strcasestr(current_desktop, "kde")) {
             const char *session_version = getenv("KDE_SESSION_VERSION");
-            if (session_version) {
+            if (session_version && *session_version > '0') {
```

Comment:
Did you mean to change the behavior here? Old behavior everything < 4 returned KDE4.